<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryGlyphsForm</class>
 <widget class="QWidget" name="GeometryGlyphsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="dataGLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="glyphDataLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Glyph Data</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="seedPointsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Seed Points   </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="glyphDataCombo"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="seedPointsCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="scaleHLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="scaleLabel">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="scaleSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tessHLayout">
     <item>
      <widget class="QLabel" name="tessLabel">
       <property name="text">
        <string>Tessellation Order</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="tessSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>6</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="glyphTypeHLayout">
     <item>
      <widget class="QLabel" name="glyphTypeLabel">
       <property name="text">
        <string>Glyph Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="glyphTypeCombo">
       <item>
        <property name="text">
         <string>3D Mesh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line Star</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="shapePage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>272</width>
        <height>451</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Shape</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="normalizeGroup">
         <property name="title">
          <string>Normalize</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="normOptionsGLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="normMethodLabel">
              <property name="text">
               <string>Method</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="normMethodCombo">
              <item>
               <property name="text">
                <string>Min/Max</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Maximum</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="normScopeLabel">
              <property name="text">
               <string>Scope</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="normScopeCombo">
              <item>
               <property name="text">
                <string>Whole Image</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Seed Points</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Local Voxel</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="sharpenGroup">
         <property name="title">
          <string>Sharpen</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="sharpenPowerHLayout">
            <item>
             <widget class="QLabel" name="sharpenPowerLabel">
              <property name="text">
               <string>Exponent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sharpenPowerSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>16.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="smoothGroup">
         <property name="title">
          <string>Smooth Glyphs</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="smoothGLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="smoothIterLabel">
              <property name="text">
               <string>Number of Iterations</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="smoothIterSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="smoothRelaxLabel">
              <property name="text">
               <string>Relaxation Factor</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="smoothRelaxSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.001000000000000</double>
              </property>
              <property name="maximum">
               <double>0.100000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
              <property name="value">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="smoothUpdateHLayout">
            <item>
             <spacer name="smoothUpdateSpacerA">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="smoothUpdateButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="smoothUpdateSpacerB">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="shapeSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="colorPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>223</width>
        <height>136</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Color</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="colorMethodHLayout">
         <item>
          <widget class="QLabel" name="colorMethodLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Method:   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="colorMethodCombo">
           <item>
            <property name="text">
             <string>Direction (RGB)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Weighted Direction (RGB)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Scalar Volume (LUT)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Radius (LUT)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="colorScalarsLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Scalar Volume</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="colorScalarsCombo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colorLUTLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Look-Up Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="colorLUTCombo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="colorSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
